app/js/main.js

```diff
@@ -74,18 +74,28 @@ var width  = 640,
 
     // Notification bar
     notifyBar    = document.querySelector(".notification"),
-    notifyBarMsg = document.querySelector(".notification #msg");
+    notifyBarMsg = document.querySelector(".notification #msg"),
+    
+    // Confirm messages
+    confirmContainer = document.querySelector("#confirm-container"),
+    confirmText      = document.querySelector("#confirm-text"),
+    confirmOK        = document.querySelector("#OK"),
+    confirmCancel    = document.querySelector("#cancel");
```

Comment:
Since IDs are obviously single-use and be accessed this way, for developer sake to help figure out where in the document this relates to, when possible I like to qualify IDs with the main parent element (even though it is technically unneeded). I'm thinking `#confirm-container #btn-cancel` (and likewise for the OK button).